eternalcore-core/src/main/java/com/eternalcode/core/feature/powertool/messages/PowertoolMessages.java

```diff
@@ -0,0 +1,13 @@
+package com.eternalcode.core.feature.powertool.messages;
+
+import com.eternalcode.multification.notice.Notice;
+
+public interface PowertoolMessages {
+
+    Notice commandAssigned();
```

Comment:
Looking at @vLuckyyy's review, I have a suggestion to call it this:

```suggestion
    Notice itemAssigned();
    Notice itemRemoved();
    Notice itemNotAssigned();
    Notice emptyCommand();
    Notice noItemInMainHand();
```

Reply:
Decided to go my own way and removed the prefixes entirely